src/preparers.py

```diff
@@ -276,46 +295,53 @@ def prepare_hc3(row, lang):
         messages.append({"from": "assistant", "text": assistant_answer, "score": 0, "parent": 0})
     return messages
 
+
 def prepare_hc3_en(row):
     return prepare_hc3(row, "en")
 
+
 def prepare_hc3_zh(row):
     return prepare_hc3(row, "zh")
 
+
 def prepare_camel_science(row):
     return convert_inputs_targets_to_messages(
         row["message_1"], row["message_2"], row["_source"],
     )
 
+
 def prepare_cot_collection(row):
     return convert_inputs_targets_to_messages(
         row["source"], row["rationale"], row['_source']
     )
 
+
 def prepare_gpt4all(row):
-    source_to_dsetid = {
-        "": "stackoverflow",
-        "pacovaldez/stackoverflow-questions": "stackoverflow",
-        "nomic-ai": "nomic",
-        "laion/unified_chip2": "chip2",
-        "unified_chip2": "chip2",
-        "unified_unifiedskg_instructions": "unifiedskg",
-        "output_unified_unifiedskg.jsonl": "unifiedskg",
-        "unified_multi_sum": "unifiedmultisum",
-        "unified_abstract_infill_output_0-100_000.jsonl": "abstractinfill",
-        "unified_abstract_infill_output-100-000-x.jsonl": "abstractinfill",
-        "unified_hc3_human": "hc3"
-    }
+    # source_to_dsetid = {
+    #    "": "stackoverflow",
+    #    "pacovaldez/stackoverflow-questions": "stackoverflow",
+    #    "nomic-ai": "nomic",
+    #    "laion/unified_chip2": "chip2",
+    #    "unified_chip2": "chip2",
+    #    "unified_unifiedskg_instructions": "unifiedskg",
+    #    "output_unified_unifiedskg.jsonl": "unifiedskg",
+    #    "unified_multi_sum": "unifiedmultisum",
+    #    "unified_abstract_infill_output_0-100_000.jsonl": "abstractinfill",
+    #    "unified_abstract_infill_output-100-000-x.jsonl": "abstractinfill",
+    #    "unified_hc3_human": "hc3"
+    # }
```

Comment:
Is this meant to be all commented out?

Reply:
I think we can delete the unused dict here, but let us create a separate PR for that and some general cleaning